example/r-template-files/README.md

```diff
@@ -0,0 +1,66 @@
+# `w-template-files`
+
+<br>
+
+While templates can be written with other code in `.ml` files, they can also
+live in their own source files.  This can be useful as templates become larger,
+or when you have many templates.
+
+If your template file is mostly HTML, you can give it a name like
+`template.eml.html`, to trigger HTML syntax highlighting by your editor.
+Additionally, if you are using `ocamlformat`, the `.html` extension will
```

Comment:
Reason users will typically use `refmt` rather than `ocamlformat`, and I haven't directly tested what will happen if you apply `refmt` to an `eml` file, or what it looks like in an editor.